script/version_check.sh

```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -Eeuxo pipefail
 
-SCRIPT_DIR=$(cd $(dirname $0); pwd)
-VERSION=$(grep '<version>' ${SCRIPT_DIR}/../pom.xml | head -n1 | sed 's/ *<[^>]*>//g')
+SCRIPT_DIR="$(cd "$(dirname "$0")" || exit 1; pwd)"
```

Comment:
I changed version_check to satisfy shellcheck. I verified locally that this script still works as intended.

Reply:
Great! Thanks!